link.x

```diff
@@ -2,21 +2,32 @@ INCLUDE memory.x
 
 SECTIONS
 {
-  .text ORIGIN(FLASH) :
+  .vector_table ORIGIN(FLASH) :
   {
     /* Vector table */
-    _VECTOR_TABLE = .;
+    _svector_table = .;
```

Comment:
this is not used in the linker script at all. It's just to make the `objdump` output slightly nicer: you get `<_svector_table>` instead of `<cortex_m_rt::RESET_HANDLER-0x4>`.